### PR TITLE
Automates packaging and (test) publishing

### PR DESCRIPTION
**Pull request by ecomba**
I modified the makefile to automate all the tasks necessary to test, package and upload the client

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,18 +15,23 @@ cd dnsimple-python
 
 Make sure you have Python (7.3 onwards) installed.
```
Comment (onlyhavecans) [inline CONTRIBUTING.md:16]: I think this version number is a typo

File: dnsimple/version.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
-version = '0.1.0'
+version = '0.1.1'
```
Comment (onlyhavecans) [inline dnsimple/version.py:1]: We should probably move to the 2.x as this is superseding the 1.x

File: CONTRIBUTING.md (hunk 1 above)
Comment (ecomba) [reply CONTRIBUTING.md:16]: You never heard of the new super-python 7.3? 🤣 

ooooooops that's actually a PHP version 🤦

Comment (ecomba) [reply CONTRIBUTING.md]: Fixed

**Pull request merged**